### DISCUSSION TITLE
Migration from old Feliz.Markdown code.

### DISCUSSION THREAD

**Discussion by EverybodyKurts** [Zaid-Ajaj/Feliz#593] [Q&A]
https://github.com/Zaid-Ajaj/Feliz/commit/684c300cb64bde7c76deae620ee56781b43c0f4c#diff-c3a91c5cc6f70deb7fd4c0f00df5e2f3aeb8109024afde2b7921f1c3a06d90edL10

Awhile ago, `language` and `value` were removed from `ICodeProperties`. What should be used in their stead?

**Comment by MangelMaxime** [comment]
First things would be to check what `react-markdown` expect you to use in term of props.

So we could look at if we have them in the binding or not. If not, we can send a PR to fix that issue.

**Comment by Zaid-Ajaj** [comment]
@EverybodyKurts I believe `className` should contain the language, although it might be in the format `language-{language}` I don't remember exactly. As for `value`, you can now just use `children: ReactElement[]` instead.
